docs/api-inference/tasks/sentence_similarity.md

```diff
@@ -0,0 +1,124 @@
+## Sentence similarity
```

Comment:
I think we wanted to remove this one in favor of feature-extraction?

Reply:
Removed it in https://github.com/huggingface/hub-docs/pull/1394/commits/c249ec64e29675a28ab1313a808c5cc885118a0d